src/endpoints/createTunewsLanguagesEndpoint.js

```diff
@@ -2,26 +2,21 @@
 
 import LanguageModel from '../models/LanguageModel'
 import EndpointBuilder from '../EndpointBuilder'
-import ParamMissingError from '../errors/ParamMissingError'
 import type { JsonTunewsLanguageType } from '../types'
 import Endpoint from '../Endpoint'
 
 export const TUNEWS_LANGUAGES_ENDPOINT_NAME = 'languages'
 
-type ParamsType = { city: ?string }
-
-export default (baseUrl: string): Endpoint<ParamsType, Array<LanguageModel>> =>
+export default (baseUrl: string): Endpoint<*, Array<LanguageModel>> =>
```

Comment:
Why `*`? Shouldn't this rather be an empty object, i.e. `{||}`?

Reply:
Actually I copied it from `SprunbrettJobs`. The `*` lets you write` tuNewsEndpoint.mapResponse(json)` instead of `tuNewsEndpoint.mapResponse(json, {})` and the same goes for `mapParamsToUrl`. Should I still replace the typing by a strict `{||}` ?

Reply:
I think its fine for me if there is a reason for it. @maxammann any other opinion?